tawazi/dag.py

```diff
@@ -324,11 +352,12 @@ def get_num_running_threads(_futures: Dict[Hashable, "Future[Any]"]) -> int:
                 num_running_threads = get_num_running_threads(futures)
                 if exec_node.is_sequential and num_running_threads != 0:
                     self.logger.debug(
-                        f"{node_id} must run without parallelisme. "
+                        f"{exec_node.id} must run without parallelisme. "
```

Comment:
I think `must not be run in parallel` would be better here

Reply:
Agree